packages/block-editor/src/hooks/layout.js

```diff
@@ -402,17 +400,7 @@ export const withLayoutStyles = createHigherOrderComponent(
 			layoutClasses
 		);
 
-		const { setStyleOverride, deleteStyleOverride } = unlock(
-			useDispatch( blockEditorStore )
-		);
-
-		useEffect( () => {
-			if ( ! css ) return;
-			setStyleOverride( id, { css } );
-			return () => {
-				deleteStyleOverride( id );
-			};
-		}, [ id, css, setStyleOverride, deleteStyleOverride ] );
+		useStyleOverride( { id, css } );
```

Comment:
This block support's id is still an instance based one. To avoid collisions should we remove `id` from this one, too?

```suggestion
		useStyleOverride( { css } );
```